tock/api/views.py

```diff
@@ -98,9 +148,19 @@ def map_row(row):
     return response
 
 def get_timecards(queryset, params={}):
```

Comment:
Again, not part of your changeset, but using mutable values like `{}` as default arguments is dangerous in python. More at http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments.